templates/emails/globalpartners/prospective_sponsor.html

```diff
@@ -7,7 +7,7 @@
 <p>We are currently looking for new corporate sponsors who will sponsor us through either monthly donations 
     or with a once off annual donation. I would like to check with you if your company would consider sponsoring Django Girls.</p>
 
-<p>Please see attached our sponsors deck for 2023 https://drive.google.com/file/d/1T2jL9tnu5ZkNvkQRAxd7jlt_BnX7VCLE/view 
+<p>Please see attached our sponsors deck for 2024 https://drive.google.com/file/d/1fAmyAiEyLGiE0s6_z3PzgLByHNKzEVHB/view
```

Comment:
@amakarudze Do these dates always want to be the current year?

To dynamically display the current year you should be able to use [`{% now "Y" %}`](https://docs.djangoproject.com/en/dev/ref/templates/builtins/#now).